pkg/app/src/components/TabPanels/Organization/tmpOverview.tsx

```diff
@@ -43,6 +48,14 @@ export function TmpOverview() {
 			if (state) {
 				// The transaction was successful, clear state
 				tmpOrgState?.clearAll()
+				result.events.forEach(({ event: { data, method, section } }) => {
+					if (section === 'control' && method === 'OrgCreated') {
+						setOrgId(data[1].toHex())
+						if (organizationByIdData && !loading) {
+							push(`/organisations/${organizationByIdData?.organization?.[0]?.id}/dashboard`)
+						}
+					}
+				})
```

Comment:
Will this work? The loading by Graph looks fine but but I think the "handleTxCallback" will be called before the organization is loaded, right?

Reply:
yes it I tried it with useEffect which will trigger when the organization is loaded